### DISCUSSION TITLE
Usage of 'Resize' in 'VideoSequential'

### DISCUSSION THREAD

**Discussion by YuhaoCheng** [kornia/kornia#928] [Questions] [answered]
I use the 'Resize' object in the 'VideoSequential' in order to transform the whole video clip, however, I encounter the error:
`torch.nn.modules.module.ModuleAttributeError: 'Resize' object has no attribute 'forward_parameters'`. So I want to know how to solve it.

Thanks.

**Comment by shijianjian** [comment]
The VideoSequential is designed for ```kornia.augmentation``` only. Also, torch can natively resize the tensor. Just fix the temporal dimension as it is.

**Comment by YuhaoCheng** [comment]
> The VideoSequential is designed for `kornia.augmentation` only. Also, torch can natively resize the tensor. Just fix the temporal dimension as it is.

Thanks for your help. I got it.  But if I want to resize the spatial dimension of the video, what should I use? For example, I get an original video with the size of 1080x1920, however, the input of the network should be 384x288. 

Thanks again.

**Comment by shijianjian** [comment, marked as answer]
This shall do:
```python
input.shape = (B, C, T, 1080, 1920)
torch.nn.functional.interpolate(input, size=(T, 384, 288))
```

**Reply by YuhaoCheng** [reply]
Thanks! I will try it. However, I have changed the pipeline of using Kornia. I will use the OpenCV to resize the frames before feeding them into Kornia's video augmentation sequential.

**Reply by shijianjian** [reply]
This is actually the recommended way of using Kornia, since Kornia is much more suitable to perform batch data than a single image. If you are interested in implementing new augmentations, feel free to send a PR.

**Comment by edgarriba** [comment]
@shijianjian maybe we could support resize3d too
